src/ensembl/src/content/app/browser/hooks/useBrowserRouting.ts

```diff
@@ -101,11 +102,14 @@ const useBrowserRouting = () => {
       const newFocus = buildFocusIdForUrl(parseEnsObjectId(activeEnsObjectId));
       dispatch(replace(urlFor.browser({ genomeId, focus: newFocus })));
     } else if (focus && !chrLocation) {
+      if (!activeGenomeId) {
+        dispatch(setActiveGenomeId(genomeId));
+      }
```

Comment:
Any chance this can be moved next to the if statement above?
https://github.com/Ensembl/ensembl-client/pull/411/files#diff-8f87f1f84e9255c291cc1bd6a5c52e16efb0474cfab2bb10053f99267b6fa994R94